### DISCUSSION TITLE
Allow to edit some properties in zotero-better-bibtex.properties

### DISCUSSION THREAD

**Discussion by faisal3325** [retorquere/zotero-better-bibtex#1792] [Ideas]
I would like to customize my auto-export git message to include branch name or what not other possibilities under my control. Can we add a provision for editing the file? Or is it already there which I am missing?

`Preferences.auto-export.git.message = {type} updated by Better BibTeX for Zotero`

**Comment by retorquere** [comment]
It's not a matter of simply changing the prompt; to include branch information I'd have to make changes to BBT, specifically, I'd have to read the output of `git branch`, but reading output from shell is very unstable in the firefox base Zotero builds on (as in it will occasionally induce a hard-crash). I could have it redirect to a temp file, but then we're looking at temp file management.

**Reply by faisal3325** [reply]
Oh, doesn't look like a straight forward solution as I thought it would be.

**Reply by faisal3325** [reply]
I created a quick githook to add additional information to the commit message:
https://gist.github.com/faisal3325/2b3efece06e9e9c7a13485948af56042

**Reply by retorquere** [reply]
hadn't thought of that, that's clever.

**Reply by faisal3325** [reply]
Thanks
